Pod/Classes/VOKCoreDataManager.h

```diff
@@ -248,10 +254,21 @@ typedef NS_ENUM (NSInteger, VOKMigrationFailureOption) {
  @param writeBlock Do not use GCD or thread jumping inside this block. Handle all fetches, creates and writes using the tempContext variable passed to this block. Do not save or merge this context, it will be done for you.
  @param completion This will be fired on the main thread once the context has been saved.
  */
-+ (void)writeToTemporaryContext:(void (^)(NSManagedObjectContext *tempContext))writeBlock
++ (void)writeToTemporaryContext:(VOKWriteBlock)writeBlock
                      completion:(void (^)(void))completion;
 
 /**
+ Deserializes the NSDictionaries full of strings in the background and creates/updates instances in the given context.
```

Comment:
I *think* I know what "NSDictionaries full of strings" means, but this could be a lot clearer